src/jit/compiler.h

```diff
@@ -4934,7 +4958,8 @@ class Compiler
         BasicBlock*     acdDstBlk; // block  to  which we jump
         unsigned        acdData;
         SpecialCodeKind acdKind; // what kind of a special block is this?
-        unsigned short  acdStkLvl;
+        unsigned        acdStkLvl;
```

Comment:
Because did not find any reasons why  'Native stack depth' can't be larger than short.
and to delete warnings/casts between this value and stkDepth/block->bbTgtStkDepth.